app/components/preprint-form-authors.js

```diff
@@ -1,7 +1,7 @@
 import Ember from 'ember';
```

Comment:
With regard to **finding/deleting components**: We should be using the ember-cli to generate stubs for all components. `ember g` and the inverse `ember d` should make this a complete non-issue. My experience is that between my git tooling and the ember-cli cleaning up after myself is really easy

I don't use a tabbed editor (or at the very least don't pay attention to tab names), so I can't really relate to difficulty finding component files. Actually pod structure does make it a bit faster for me to work on a component's controller/template logic, but the difference here is pretty negligible. 

> Pods do have some major drawbacks; we had to do some hoop jumping to make ember-osf recognize style files outside the styles/ folder. It's genuinely annoying, but a solvable problem.

IMO this is an unanswered question for _Ember in general_, i.e. totally indep. of code structure. There's no difficulty using structure for components and still packaging component styles in `/app/styles/components/`. But like Andy suggests, we absolutely should keep component styles in separate files. 

Sorry there is nothing very definitive here. I do prefer pod styles-- the semantics of bundling controller/template/style logic in a single directory really appeals to me. There is a lot to be said for consistency in the codebase, and I do think the Ember community is moving this direction (favoring pod structure). But I can't say one way is right or wrong.